package.json

```diff
@@ -67,8 +67,9 @@
     "@strapi/admin-test-utils": "workspace:*",
     "@strapi/eslint-config": "0.1.2",
     "@swc/cli": "0.1.62",
-    "@swc/core": "1.3.37",
-    "@swc/jest": "0.2.24",
+    "@swc/core": "1.3.58",
+    "@swc/helpers": "0.5.1",
```

Comment:
Explicitly added helpers... Cause otherwise it seems to fallback to an older version from

https://github.com/adobe/react-spectrum/blob/dd60f3969171a80a86851cf48011da9bf3290b6e/packages/%40internationalized/number/package.json#L25

I'll open a P/R upstream as soon as I confirm

Reply:
Could you elaborate a bit what @swc/helpers is for? As for the rest the documentation is very sparse 🙈

Reply:
Like babel/helpers or Tslib to some extend. Swc/core and helpers works together. AFAIK when there's mismatches between their versions,  the thing becomes buggy. I guess will be solved with v1.

That's all I know